lib/acm_bot/cache.ex

```diff
@@ -0,0 +1,30 @@
+defmodule AcmBot.Cache do
+  use GenServer
+  require Logger
+
+  # Client API
+  def start_link(args) do
+    GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
+  end
+
+  def set_file_id(id) do
+    GenServer.cast(__MODULE__, {:set_file_id, id})
+  end
+
+  def get_file_id() do
+    GenServer.call(__MODULE__, :get_file_id)
+  end
+
+  # Server Callbacks
+  def init(:ok) do
+    {:ok, ""}
```

Comment:
A cache that only saves an ID it's a little weird. What would happen if you need to save another file ID here? Would you create 2 AcmBot.Cache, one for every ID?

The typical way of doing this is having the inner state as a map, and accessing the cache with a key, for example:
`AcmBot.Cache.get(:file_id_comida)` and `AcmBot.Cache.put(:file_id_comida, file_id)`